docs/pages/access-controls/device-trust/guide.mdx

```diff
@@ -4,14 +4,14 @@ description: Get started with Teleport Device Trust
 videoBanner: gBQyj_X1LVw
 ---
 
-<Admonition type="warning">
+<Admonition type="notice">
   Device Trust supports the following components:
 
   - User devices: macOS, Windows and Linux.
   - Teleport client: `tsh` and Teleport connect.
-  - Resources: SSH, Database and Kubernetes.
+  - Resources: Apps (role-based only), SSH, Database and Kubernetes
```

Comment:
```suggestion
  - Resources: apps (role-based only), SSH nodes, databases, and Kubernetes clusters.
```
To make this a list of resources

Reply:
Also, I'm not sure the docs talk about what a role-based app is elsewhere—is there another wording we can use for this?

Reply:
How about "role-based enforcement only" ??

Reply:
Applied in both instances, used "role-based enforcement only".